packages/core/src/zig/lib.zig

```diff
@@ -1437,3 +1438,142 @@ export fn bufferDrawChar(
     const rgbaBg = utils.f32PtrToRGBA(bg);
     bufferPtr.drawChar(char, x, y, rgbaFg, rgbaBg, attributes) catch {};
 }
+
+// Windows Console Control Handler
```

Comment:
Cool! Can we just have that in an extra module? The `lib.zig` is getting convoluted. It could lazily set that up once when the firs renderer is created maybe?